lib/tabula.ex

```diff
@@ -61,6 +62,11 @@ defmodule Tabula do
     def keys(row), do: row |> Keyword.keys
   end
 
+  defimpl Row, for: Any do
+    def get(row, col, default \\ nil), do: row |> Map.get(col, default)
```

Comment:
Here I am @aerosol I totally agree with your suggestion.

I modified the code as you suggested.

Do you think I should add a test for it?